test/verify/check-machines

```diff
@@ -107,6 +107,8 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             m.execute("until test -e /run/libvirt/libvirt-sock; do sleep 1; done")
             m.execute("chmod o+rwx /run/libvirt/libvirt-sock")
 
+        m.execute("virsh net-define /etc/libvirt/qemu/networks/default.xml")
```

Comment:
Nice! I looked into that a while ago, but couldn't figure out what undefines it. Is that idempotent?

Reply:
We do undefine it in some tests - which might fail and then don't redefine later. 
I think it's totally fine to define it in the start of the test.
And yes it's indempotent.

Reply:
Seems to work.